core/src/main/res/layout/mz_view_map.xml

```diff
@@ -30,16 +30,46 @@
       android:layout_width="@dimen/mz_find_me_button_width"
       android:layout_height="@dimen/mz_find_me_button_height"
       android:layout_alignParentRight="true"
-      android:layout_alignParentBottom="true"
+      android:layout_alignParentTop="true"
       android:layout_marginRight="@dimen/mz_find_me_button_margin_right"
-      android:layout_marginBottom="@dimen/mz_find_me_button_margin_bottom"
+      android:layout_marginTop="@dimen/mz_find_me_button_margin_top"
       android:background="@drawable/mz_bg_white_gray_border"
       android:visibility="gone"
       android:src="@drawable/mz_find_me"
       android:contentDescription="@string/mz_find_me"
       tools:ignore="RtlHardcoded"
       />
 
+  <ImageButton
+      android:id="@+id/mz_zoom_in"
```

Comment:
Yeah I tried the same to no effect.

Oh well since this issue is only cosmetic I'm happy to merge as-is and we can come up with a fix before the next release.

Thanks again for the contribution!